hazelcast-client/src/test/java/com/hazelcast/client/listeners/AbstractListenersOnReconnectTest.java

```diff
@@ -585,9 +581,6 @@ private ClientConfig getNonSmartClientConfig() {
     void onEvent(String event) {
         events.remove(event);
         eventCount.incrementAndGet();
-        if (events.isEmpty()) {
-            eventsLatch.countDown();
```

Comment:
I removed the latch because no one setting it to 1 again. In case `outer assertTrueEventually` cause the verify part run again, it was working wrong. 
You are right, I forget about its use and since I already removed `outer assertTrueEventually`  I can safely put it back.